v1/content/getting-started/docker.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Docker"
+weight: 3
+---
+
+### Images
+
+Docker images of ThingsDB are available at [GitHub](https://github.com/thingsdb/ThingsDB/pkgs/container/node/).
+
+**Supported tags**:
+
+`ghcr.io/thingsdb/node:VERSION` _(Minimal ThingsDB image based on Alpine Linux.)_
+
+`ghcr.io/thingsdb/node:gcloud-VERSION` _(Includes Google Cloud Utilities so ThingsDB can created backups in the Google Cloud.)_
+
+`ghcr.io/thingsdb/node:full-VERSION` _(Based on a Debian image with Google Cloud Utilities, Python3 and [py-timod](https://pypi.org/project/py-timod/) installed.)_
+
+`ghcr.io/thingsdb/node:latest` _(Latest ThingsDB build from the `master` branch using a minimal Alpine Linux base image)_
+
+### Run ThingsDB in GKE (Google Kubernetes Engine)
```

Comment:
Shouldn't we move this to the "main" getting started as a fourth option after Docker? It is not really Docker, it is Kubernetes which happens to use our default docker images as an example.